.yamllint

```diff
@@ -5,6 +5,7 @@ extends: default
 ignore: |
   molecule/**/tests/
   node_modules
+  .travis.yml
```

Comment:
Beign this file a yaml... Should we really exclude it from yamllint?

Reply:
this file was generated by travis client and it's a mess for the linter so I prefer to exclude it :)